python/lsst/ts/standardscripts/maintel/home_both_axes.py

```diff
@@ -56,9 +56,9 @@ class HomeBothAxes(salobj.BaseScript):
     def __init__(self, index, add_remotes: bool = True):
         super().__init__(index=index, descr="Home both TMA axis.")
 
+        self.disable_m1m3_force_balance = False
         self.home_both_axes_timeout = 300.0  # timeout to home both MTMount axes.
 
-        self.ignore_m1m3 = False
         self.warn_wait = 10.0
```

Comment:
I think you can remove this attribute now.

Reply:
It is removed already, see the "-"

Reply:
I mean the `warn_wait`....